src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets

```diff
@@ -29,37 +29,67 @@ Copyright (c) .NET Foundation. All rights reserved.
     automatically selects the version of .NET Core to target.
 
     The framework version that is written to the runtimeconfig.json file is based on the actual resolved package version
-    of Microsoft.AspNetCore.App or Microsoft.AspNetCore.All. This is to allow floating the verion number (i.e. the user
+    of Microsoft.AspNetCore.App or Microsoft.AspNetCore.All. This is to allow floating the version number (i.e. the user
     could set the version to "2.0-*".
 
   -->
   <Choose>
     <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" >
 
+      <!-- 
+        These properties will allow the default and latest patch versions to be set in the CLI (via BundledVersions.props) 
+        or overridden by tests. The latest patch version for 2.1 uses a while card so self-contained deployment gets the 
+        latest stable metapackage.
+      -->
+      <PropertyGroup>
+        <DefaultPatchVersionForAspNetCoreAll2_1 Condition="'$(DefaultPatchVersionForAspNetCoreAll2_1)' == ''">2.1.1</DefaultPatchVersionForAspNetCoreAll2_1>
+        <DefaultPatchVersionForAspNetCoreApp2_1 Condition="'$(DefaultPatchVersionForAspNetCoreApp2_1)' == ''">2.1.1</DefaultPatchVersionForAspNetCoreApp2_1>
+        
+        <LatestPatchVersionForAspNetCoreAll2_1 Condition="'$(LatestPatchVersionForAspNetCoreAll2_1)' == ''">2.1.*</LatestPatchVersionForAspNetCoreAll2_1>
```

Comment:
@dsplaisted how do you plan to handle the latest patch version for NETCore.App 2.1.x in the 2.2 SDK?

Reply:
We have a [hard-coded list](https://github.com/dotnet/cli/blob/80d542b8f4eff847a0f72dc8f2c2a29851272778/build/MSBuildExtensions.targets#L174-L176) of latest patch versions, and then [tests](https://github.com/dotnet/cli/blob/master/test/EndToEnd/GivenSelfContainedAppsRollForward.cs) which verify that those versions are correct so that we remember to keep them up-to-date.  I'd recommend doing the same thing for the ASP.NET latest patch versions.  https://github.com/dotnet/cli/pull/9374 puts the default versions in the CLI, I would also add the latest versions and tests to make sure they're up-to-date.

Reply:
Yes, we should do this. I've opened https://github.com/dotnet/cli/issues/9403 to follow-up on this.